components/molecule/imageEditor/src/index.js

```diff
@@ -6,21 +6,17 @@ import {debounce} from '@s-ui/js/lib/function/debounce.js'
 import Injector from '@s-ui/react-primitive-injector'
 
 import getCroppedImg from './utils/cropImage.js'
-import ImageEditorDefault from './ImageEditorDefault.js'
+import {baseClass, DEFAULT_ASPECT, getRotationDegrees, noop} from './config.js'
 import ImageEditorCropper from './ImageEditorCropper.js'
+import ImageEditorDefault from './ImageEditorDefault.js'
 import ImageEditorSliders from './ImageEditorSliders.js'
-import {
-  baseClass,
-  DEBOUNCING_TIME,
-  DEFAULT_ASPECT,
-  noop,
-  getRotationDegrees
-} from './config.js'
+import {debouncingTime} from './prop-types.js'
 
 const MoleculeImageEditor = ({
   aspect = DEFAULT_ASPECT,
   cropLabelIcon,
   cropLabelText,
+  debouncingTime,
```

Comment:
given this is clearly a performance improvement, shouldn't we set a default value for this prop?

Reply:
not needed, if it's not declares we will recover the previous version. debounced on demand

Reply:
I'm more concerned about it from a functional standpoint. If the component without a debounce time is not performing correctly, shouldn't we provide a default value that makes the component work _by default_?